CoinJoin.md

```diff
@@ -0,0 +1,129 @@
+
+CoinJoin is a Bitcoin-only protocol. The way the use of it differs depends on the wallet/mixer that you are using. Some of the most known privacy-focused Bitcoin wallets are Wasabi, Samourai. Let's go through some details concerning anonymity sets and fees for each of them:
+
+WabiSabi Wallet
+
+The WabiSabi Wallet charges a coordinator fee of 0.3% for CoinJoin transactions of more than 0.01 BTC. The coordinator fee is charged to cover the service costs of coordinating the mix. For amounts of 0.01 BTC or below, no coordinator fee is charged.
+
+An anonymity set is basically the pool of users participating in a specific transaction. The size of an ideal anonymity set in a WabiSabi can vary from 50 to 100 participants in a CoinJoin round. This means your transaction would be mixed with those 49 to 99 other users.
+
+Samourai 
+
+Samourai Wallet uses Whirlpool, a CoinJoin implementation created by its developer team. The service employs a fixed fee structure. Fees range from 5,000 satoshis for the smallest pool (100,000 satoshis) to 1,750,000 satoshis for the largest pool (50,000,000 satoshis). Each satoshi is equal to one hundred millionth of a Bitcoin (0.00000001 BTC), it is the smallest Bitcoin unit.
+
+Whirlpool's anonymity sets vary based on real-time participation within each pool. Since the number fluctuates the user is encouraged to go through multiple mixing rounds to increase their transaction's anonymity. Samourai wallet model aims to provide a balance between costs and privacy.
+
+Bitcoin mixers
+
+There are also options for those wanting a fast and user-friendly Bitcoin mixer. Websites like Whir and Tumber.io let you mix your Bitcoin by entering the Bitcoin address where you want to receive the mixed BTC. After that you need to send the BTC to the address generated by the tool, they will be sent to a BTC pool, where your Bitcoin will be mixed, and after that sent to the address you entered in the first step.
+
+Whir has a 1% fixed coordinator fee.
+Tumber.io fees start at 0.4% and vary according to the amount mixed.
+
+## Could it work for Decred?
+
+As we said before, CoinJoin is a tool exclusive to Bitcoin. Other coin mixers have been developed by other projects, like the Stake Shuffle protocol from Decred.
+
+StakeShuffle is a privacy protocol available at Decred Wallet Decrediton. It is a non-custodial process used to obfuscate ownership of DCR coins. If you use StakeShuffle your outputs are going to turn fully anonymized, and peers of the server won't be able to link outputs and inputs. This technology is based on a paper called " “P2P Mixing and Unlinkable Bitcoin Transactions” by Ruffing, Moreno-Sanchez, and Kate. This system uses a more efficient version known as DiceMix Light, making the process quicker. Although it successfully hides who is sending money to whom, the amounts being sent are still visible to everyone. For added privacy, each mixing transaction uses the same amount of money, known as a fixed denomination.
```

Comment:
It's probably worth mentioning that Decred's stakeshuffle is a massive upgrade to the conjoin protocol originally caller coinjoin++. It also has some unique properties specific to the Decred project e.g. it combines with the ticket pool to creates an extremely large anonymity set, something in the realms of 41,000 participates at any given time and a pool of coins over 60% of the supply.

Reply:
It might also be worth mentioning that the bulk of Decred's mixing is processed off-chain meaning it doesn't cause unnecessary bloat to the chain. Also the devs are currently working on including the mixing service / server as part of dcrd, making it a core feature of running a full node.

Reply:
didn't know that. Thanks i will add it